aea/cli/run.py

```diff
@@ -279,7 +310,9 @@ def run_aea(
     )
 
     click.echo(AEA_LOGO + "v" + __version__ + "\n")
-    _print_hash_table(ctx, aea)
+    _print_all_available_packages(ctx)
+    _print_instantiated_components(aea)
```

Comment:
We first print a table of all available packages and then a table of components that were instantiated.